tests/src/unit_v2_tests.cpp

```diff
@@ -0,0 +1,197 @@
+#include <cstdint>
+#include <ratio>
+#include <type_traits>
+
+#include <gtest/gtest.h>
+#include <stronk/stronk.h>
+#include <stronk/unit_v2.h>
+#include <stronk/utilities/dimensions.h>
+
+namespace twig::unit_v2::unit_tests
+{
+
+// Example code:
+
+// First we define our stronk types:
+struct Distance : unit<Distance>
+{
+    template<typename T>
+    struct value
+        : stronk<value<T>, T, can_equate_with_is_close>
+        , value_of_unit<Distance>
+    {
+        using stronk_base_t = ::twig::stronk<value<T>, T, can_equate_with_is_close>;
+        using stronk_base_t::stronk_base_t;
+    };
+};
+template<typename T>
+using distance_t = Distance::template value<T>;
```

Comment:
Hmm - that's a lot of code to define a type.

I guess you expect to gain some flexibility from that, but maybe we can create a helper template for this?

Reply:
Yea I tried to think would you could do it. In general it's only 4 lines more (3 for the wrapper class and one more because stronk::stronk does not work :(

Reply:
Ill at least make a helper make<unit, underlying>

Reply:
And Ill try to see what I can do about reducing